src/object/patch/rm-link.js

```diff
@@ -120,33 +44,26 @@ module.exports = (createCommon, options) => {
 
       expect(withoutChildCid).to.not.deep.equal(parentCid)
       expect(withoutChildCid).to.deep.equal(nodeCid)
+
+      /* TODO: revisit this assertions.
```

Comment:
Can this be uncommented?

Reply:
Same here: https://github.com/ipfs/interface-js-ipfs-core/pull/547#discussion_r342693343